src/pages/MyProfilePage.js

```diff
@@ -0,0 +1,37 @@
+/** @format */
```

Comment:
- [ ] Kindly make sure you do not have any console errors that are displayed. Currently, you do have two errors.


> WHEN I GO TO ROCKETS
> <img width="689" alt="Screen Shot 2022-08-26 at 06 28 02" src="https://user-images.githubusercontent.com/69011963/186797091-a76739ab-d58c-4a65-8bd3-5b7f2cd08b10.png">
> 
> WHEN I GO TO MY MISSIONS
> <img width="686" alt="Screen Shot 2022-08-26 at 06 28 11" src="https://user-images.githubusercontent.com/69011963/186797111-5a16d939-d426-42b1-b22e-31393596aed9.png">
> 
>